src/bitcoin-cli.cpp

```diff
@@ -568,20 +581,21 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "Returns a network peer connections dashboard with information from the remote server.\n"
         "This human-readable interface will change regularly and is not intended to be a stable API.\n"
         "Under the hood, -netinfo fetches the data by calling getpeerinfo and getnetworkinfo.\n"
-        + strprintf("An optional integer argument from 0 to %d can be passed for different peers listings; %d to 255 are parsed as %d.\n", MAX_DETAIL_LEVEL, MAX_DETAIL_LEVEL, MAX_DETAIL_LEVEL) +
+        + strprintf("An optional integer argument from 0 to %u can be passed for different peers listings; %u to 255 are parsed as %u.\n", NETINFO_MAX_DETAIL_LEVEL, NETINFO_MAX_DETAIL_LEVEL, NETINFO_MAX_DETAIL_LEVEL) +
         "Pass \"help\" to see this detailed help documentation.\n"
         "If more than one argument is passed, only the first one is read and parsed.\n"
         "Suggestion: use with the Linux watch(1) command for a live dashboard; see example below.\n\n"
         "Arguments:\n"
-        + strprintf("1. level (integer 0-%d, optional)  Specify the info level of the peers dashboard (default 0):\n", MAX_DETAIL_LEVEL) +
+        + strprintf("1. level (integer 0-%u, optional)  Specify the info level of the peers dashboard (default 0):\n", NETINFO_MAX_DETAIL_LEVEL) +
         "                                  0 - Connection counts and local addresses\n"
         "                                  1 - Like 0 but with a peers listing (without address or version columns)\n"
         "                                  2 - Like 1 but with an address column\n"
         "                                  3 - Like 1 but with a version column\n"
         "                                  4 - Like 1 but with both address and version columns\n"
+        "                                  5 - Like 4 but with additional data for development and testing purposes\n"
```

Comment:
The help messages for the other levels give you insight into what you'll get before you use the level. This new help message for level 5 is kind of like a mystery box for users who are not familiar with the netinfo code -- Look inside for additional data 🎁

At this level we are introducing two new data fields `addrl` and `addrp` which are related to addresses and rate-limiting. I think at minimum we could just say that the additional data is address data, now it's not such a mystery :)

```suggestion
        "                                  5 - Like 4 but with additional address data for development and testing purposes\n"
```

Reply:
Dropped the additional level (thanks!)